packages/strapi-plugin-sentry/services/sentry.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const Sentry = require('@sentry/node');
+const defaultSettings = require('../config/settings.json');
+
+module.exports = {
```

Comment:
Something you can do to avoid having to use this and the issues it causes is the following:


```js
const createSentryService = () => {
  let isReady = false;
  let instance = null;

  const settings = {}; 
 
  return {
    init() { /* ... */},
    getInstance() {},
    sendError() {}
  }
}

module.exports = createSentryService();
```